README.md

```diff
@@ -12,50 +22,50 @@ Platform support:
 - WebAssembly
 - Linux (Skia.Gtk)
 
-![Uno Material](./images/UnoMaterial.png)
+## Packages
 
-<!-- TODO : Insert build status -->
 Package|Stable|Preview
 -|-|-
 Uno.Material|[![NuGet stable](https://img.shields.io/nuget/v/Uno.Material?label=stable)](https://www.nuget.org/packages/Uno.Material)|[![NuGet preview](https://img.shields.io/nuget/vpre/Uno.Material?label=preview)](https://www.nuget.org/packages/Uno.Material)
 Uno.Material.WinUI|[![NuGet stable](https://img.shields.io/nuget/v/Uno.Material.WinUI?label=stable)](https://www.nuget.org/packages/Uno.Material.WinUI)|[![NuGet preview](https://img.shields.io/nuget/vpre/Uno.Material.WinUI?label=preview)](https://www.nuget.org/packages/Uno.Material.WinUI)
 Uno.Cupertino|[![NuGet stable](https://img.shields.io/nuget/v/Uno.Cupertino?label=stable)](https://www.nuget.org/packages/Uno.Cupertino)|[![NuGet preview](https://img.shields.io/nuget/vpre/Uno.Cupertino?label=preview)](https://www.nuget.org/packages/Uno.Cupertino)
 Uno.Cupertino.WinUI|[![NuGet stable](https://img.shields.io/nuget/v/Uno.Cupertino.WinUI?label=stable)](https://www.nuget.org/packages/Uno.Cupertino.WinUI)|[![NuGet preview](https://img.shields.io/nuget/vpre/Uno.Cupertino.WinUI?label=preview)](https://www.nuget.org/packages/Uno.Cupertino.WinUI)
 
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](LICENSE)
-
 
 ## Getting Started
 
-See the complete [Getting Started](doc/getting-started.md) guide for starting with this library.
+See the complete [documentation](#documentation) for starting with this library.
 
 For a larger example and features demo, visit the [Uno Gallery](https://github.com/unoplatform/uno.gallery) repository.
 
 ## Documentation
 
-All documentation for Uno.Themes can be found [here](doc/) or on [Uno Platform documentation](https://platform.uno/docs/articles/intro.html).
-
-## License
-
-This project is licensed under the Apache 2.0 license - see the
-[LICENSE](LICENSE) file for details.
+All documentation for `Uno.Themes` can be found on our [website](https://platform.uno).
```

Comment:
This will be updated to point to the actual Themes section of the docs on the main site when its ready right?

Reply:
Yes @kazo0, but I think we will wait until the end to update it. It's easy to update the readme at the last minute and this file is not related to the main doc publication.